src/components/structures/ThreadView.tsx

```diff
@@ -353,7 +353,7 @@ export default class ThreadView extends React.Component<IProps, IState> {
                     ref={this.timelinePanel}
                     showReadReceipts={true}
                     manageReadReceipts={true}
-                    manageReadMarkers={true}
+                    manageReadMarkers={false}
```

Comment:
I was wondering what this actually did, but the comment just says "Enable managing RRs and RMs." Well gee thanks.

What does *managing* actually entail? and why do we now want to turn it off?